### PR TITLE
[CPU][ARM] Reshape tensors for ACLConvertExecutor

### PR DESCRIPTION
**Pull request by alvoron**
TODO:
- [x] add 7D convert test

### CODE REVIEW

**Comment by dmitry-gorokhov** [comment]
@allnes please review the PR

**Review by allnes — APPROVED** [review]

**Review by riverlijunjie — APPROVED** [review]
LGTM, maybe add some tests for 8D and 10D.

File: src/plugins/intel_cpu/src/nodes/executors/acl/acl_utils.hpp [hunk 1]
```diff
@@ -10,6 +10,28 @@
 namespace ov {
 namespace intel_cpu {
 
+/**
+* @brief ACL supports arm_compute::MAX_DIMS maximum. The method squashes the last
+* dimensions in order to comply with this limitation
+* @param dims vector of dimensions to squash
+* @return vector of dimensions that complies to ACL
+*/
+inline VectorDims reshape_sizes(VectorDims dims) {
```
Comment (dmitry-gorokhov) [inline src/plugins/intel_cpu/src/nodes/executors/acl/acl_utils.hpp:19]: I would say this is not self-explainable name. I can propose smt like: "collapse_dims_to_max_rank".

**Comment by dmitry-gorokhov** [comment]
@alvoron Please extend tests as we agreed offline

**Comment by dmitry-gorokhov** [comment]
@alvoron Is the PR finilized and can be reviewed?

**Comment by alvoron** [comment]
> @alvoron Is the PR finilized and can be reviewed?

Yes, please.

File: src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/arm/conversion.cpp [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+using namespace InferenceEngine;
+using namespace CPUTestUtils;
+using namespace ngraph::helpers;
+using namespace ov::test;
+
+namespace CPULayerTestsDefinitions {
+namespace Conversion {
+
+INSTANTIATE_TEST_SUITE_P(smoke_ConvertCPULayerTest_7D_Dynamic, ConvertCPULayerTest,
```
Comment (dmitry-gorokhov) [inline src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/arm/conversion.cpp:17]: Why no to make these instances common? I expect it should work correctly on any target.

Comment (alvoron) [reply src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/arm/conversion.cpp:17]: These 7D tests go to the x86 reference implementation.

**Review by dmitry-gorokhov — APPROVED** [review]

**Pull request merged**